plugins/CoreAdminHome/lang/en.json

```diff
@@ -179,6 +179,7 @@
         "NeedHelp": "Need help?",
         "HowToGetHelp": "There are different ways you can get help. There is free support via the Matomo Community and paid support provided by the Matomo team and partners of Matomo. Or maybe do you have a bug to report or want to suggest a new feature?",
         "GoogleTagManagerDescription": "If your website is using Google Tag Manager, please see %1$sthis guide%2$s.",
-        "WordpressDescription": "If your website is using Wordpress, please see %1$sthis guide%2$s."
+        "WordpressDescription": "If your website is using Wordpress, please see %1$sthis guide%2$s.",
+        "ReactDescription": "If your website is using React Js, please see %1$sthis guide%2$s."
```

Comment:
The library is called `React`, alternative names could be `React.js` or `ReactJS`. We should pick one and stick with it consistently across our messaging and docs. `React` is the correct way.

```suggestion
        "ReactDescription": "If your website is using React, please see %1$sthis guide%2$s."
```

It's also a question which is the best way how to phrase this — is using React, is built with React, is a React app etc.